libraryBuilds/prep.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+J2OBJC_VERSION=0.9.8.1
+mkdir localJ2objcDist
+mkdir common
+
+# Fail if any command fails
+set -ev
+
+pushd localJ2objcDist
+
+# For developer local testing, don't keep redownloading the zip file.
+if [ ! -e j2objcDist.zip ]; then
```

Comment:
Won't this fail when J2OBJC_VERSION is changed? I think this doesn't matter as a developer should know to delete the old version... while on the continuous build, it's blown away each time?